### ISSUE TITLE
Version 2.104.2: PHP Parse error:  syntax error, unexpected ')' in src/Pdf/FPDIWriter.php on line 154

### ISSUE THREAD

**Issue by homberger** [heimrichhannot/contao-utils-bundle#11]
```
> Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication
PHP Parse error:  syntax error, unexpected ')' in myproject/vendor/heimrichhannot/contao-utils-bundle/src/Pdf/FPDIWriter.php on line 154
Script Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication handling the post-update-cmd event terminated with an exception


  [RuntimeException]
  An error occurred while executing the "contao:install-web-dir" command: PHP Parse error:  syntax error, unexpected ')' in myproject/vendor/heimrichhannot/contao-utils-bundle/src/Pdf/FPDIWriter.php on line 154
```

**Comment by koertho** [comment]
@homberger This should be fixed now in 2.104.3, can you please report back?

**Comment by homberger** [comment]
>This should be fixed now in 2.104.3, can you please report back?

`Updating heimrichhannot/contao-utils-bundle (2.104.2 => 2.104.3)`

Fix verified for 2.104.3! Thank you! :)
